src/main/webapp/i18n/en/tutorialGroupImport.json

```
@@ -6,19 +6,19 @@
             "importButton": "Import Tutorial Groups",
             "dialogTitle": "Import Tutorial Group Registrations",
             "explanations": {
-                "description": "This dialog can be used to import tutorial groups and their registrations from a CSV file. Tutorial Groups will be created if they do not exist yet. Students will be deregistered from their current tutorial group and registered in the new one.",
-                "requirements": "Each row must always contain a value for the tutorial group title. If you want to not only create tutorial groups but also register students, the row must contain a value for either matriculation number or login. This is needed to identify the student. Values for last and first name are optional. The top row must specify the header!"
+                "description": "This dialog allows you to import tutorial groups and student registrations from a CSV file. If a tutorial group does not exist, it will be created. If students are listed in the CSV then they will be moved from their current tutorial group to the new one specified in the CSV file.",
+                "requirements": "The top row of the CSV file must specify the headers. Each row must include a title for the tutorial group. To register students, each row must additionally include either the matriculation number or the login of the student. First and last names are optional."
             },
             "supportedHeaderNames": {
                 "label": "Supported header names (versions with spaces, underscores, hyphens and uppercase letters are also supported): ",
-                "alwaysRequired": "Always required (to identify tutorial group): ",
-                "requiredIfStudentRegistration": "One of those is required if you want to register students (to identify students) :",
-                "alwaysOptional": "Always optional (just for display purposes): ",
-                "tutorialGroupTitle": "For tutorial group title: {{ headers }}",
-                "registrationNumber": "For matriculation number: {{ headers }}",
-                "login": "For login: {{ headers }}",
-                "firstName": "For first Name: {{ headers }}",
-                "lastName": "For last Name: {{ headers }}"
+                "alwaysRequired": "One of these Header is always required to identify the tutorial group: ",
+                "requiredIfStudentRegistration": "One of these is required as Header if you want to identify and register students :",
```

```suggestion
                "requiredIfStudentRegistration": "One of these is required as Header if you want to identify and register students:",
```

I think there shouldn't be an empty space there

---

fixed 👍